camel/memories/blocks/chat_history_block.py

```diff
@@ -86,11 +84,11 @@ def get_context(self) -> Tuple[List[OpenAIMessage], int]:
                 output_records.append(ContextRecord(record, 1.0))
             else:
                 # Other messages' score drops down gradually
-                score *= 0.99
+                score *= 0.9
```

Comment:
the fixed rate (score *= 0.9) might be replaced or supplemented with more complex logic based on other factors (time stamps, message importance, etc.), how about allow customization of this rate?